pkg/runtime/setup.go

```diff
@@ -40,6 +41,11 @@ type Opts struct {
 	EventHandlers        []events.HandlerFunc
 	BuildlogFilePath     string
 
+	// Options for setting up a runtime from an archive.
+	FromArchiveDir string
+	PlatformID     *strfmt.UUID
+	ArtifactExt    string
```

Comment:
Please make a struct to put these under. As they are all directly related and should never be used independently from one another.